app/models/user.rb

```diff
@@ -24,9 +24,8 @@ class User < ActiveRecord::Base
   has_many :project_roles, through: :identity_group
   has_many :collection_roles, through: :identity_group
 
-  validates :login, presence: true, uniqueness: true
-  validates_presence_of :display_name
-  validates_uniqueness_of :display_name, case_sensitive: false
+  validates :display_name, presence: true, uniqueness: { case_sensitive: false },
+            format: { without: /\$|@|\s+/ }
```

Comment:
Should we extract the shared User/UserGroup display_name regex to a common constant to decrease the chance of them diverging?